src/std/text.ab

```diff
@@ -142,8 +142,26 @@ pub fun char_at(text: Text, index: Num): Text {
 }
 
 /// Capitalize the first letter of the given `text`.
+#[allow_absurd_cast]
 pub fun capitalize(text: Text): Text {
```

Comment:
Please don't mix `if ... :` and `if ... { }` syntax. I suggest the following:
```
/// Capitalize the first letter of the given `text`.
#[allow_absurd_cast]
pub fun capitalize(text: Text): Text {
    trust {
        if len(text) == 0 {
            return text
        }
        const bash_version = $ echo \"\$\{BASH_VERSINFO[0]}\" $ as Num
        if bash_version >= 4 {
            return $ echo \"\$\{text^}\" $
        }
        // GNU sed supports \U
        $ re='\bCopyright\b.+\bFree Software Foundation\b'; [[ \$(sed --version 2>/dev/null) =~ \$re ]] $
        if status == 0 {
            return $ echo "{text}" | sed "s/^\(.\)/\U\1/" $
        }
        let first_letter = upper(char_at(text, 0)) 
        return first_letter + slice(text, 1)
    }
}
```

Reply:
Thank you, done!